index.html

```diff
@@ -1,82 +1,112 @@
+    <h2 class='heading'>Tipser Collection:</h2>
+    <!-- This is a marker element for tipser Collection, given tipser collection id -->
+    <p data-tipser-cid="5cf51fec561f17000129fcb7" data-tipser-imgsize="1"><a
+            href="https://www.tipser.com:443/shops/tipser/shoes"></a>SHOES</a> </p>
 
-    <h2 class="heading">Collection (1 element)</h2>
+    <h2 class="heading">Collection: (1 element)</h2>
     <div data-tipser-cid="5cf524488643910001afc41a"></div>
 
-    <h2 class="heading">Store</h2>
+    <h2 class="heading">Store:</h2>
     <!-- This is a marker element for tipser shop -->
     <div id="tipser_shop"></div>
 
-    <h2 class="heading">Store  - square tiles</h2>
-    <!-- This is a marker element for tipser shop -->
-    <div id="tipser_shop" class="square-tiles"></div>
-
-    <h2 class="heading">Product compact - custom styles</h2>
-    <p style="text-align: center">Example of custom styles attached to the project, that override Product default styles.
-        <span class="code">.custom-styles .te-product-card.te-product-card--compact {font-size: 20px; max-width: 400px;} </span>
-
-        <span class="code">.custom-styles .te-product-card.te-product-card--compact .te-product-card-img {height: 400px;}</span>
-
-        <span class="code">.custom-styles .te-product-card.te-product-card--compact .te-product-card-brand {color: #5F9F9F;}</span>
-
-        <span class="code">.custom-styles .te-product-card.te-product-card--compact .te-product-card-price {color: maroon;}</span>
-    </p>
-    <!-- This is a marker element for tipser product with compact mode enabled, given tipser product id -->
-    <div data-tipser-pid="5a178b7a8aa0cd37ec7983e7" data-tipser-view="compact" class="custom-styles"></div>
-
-
-
     <script>
         var scriptTag = document.createElement('script');
         scriptTag.src = "https://cdn.tipser.com/tipser-script/dev.js";
         // in case you want to load local dev version of script
         // scriptTag.src = "http://127.0.0.1:8080/Script-tipser-elements.js";
 
-        scriptTag.onload = scriptTag.onreadystatechange = function() {
+        scriptTag.onload = scriptTag.onreadystatechange = function () {
             tipser.elements('59e86b79b8f3f60a94ecd26a', { //pos id (Tipser user id)
-                    //The configuration of tipser widget comes here
-                    env: "dev",
-                    lang: "se-SV",
-                    eventsHandlers: {
-                        onError: function(error){
-                            console.error(error)
-                        },
-                        onAddToCart: function(options) {
-                            console.log('Hurray, you have added item to cart. ', options.product);
-                            console.log('Your cart size is now. ', options.cartSize);
-                        }
-            }
-                })
+                //The configuration of tipser widget comes here
+                env: "dev",
```

Comment:
According to the task `env` is supposed to be changed to `'prod'`.

Reply:
done